### PR TITLE
DEP: remove parameter centered of stats.qmc.LatinHypercube

### LINKED ISSUE #18582: DEP: removal of parameter `centered` of `stats.qmc.LatinHypercube`
Follow-up of #17152

### PR DESCRIPTION
**Pull request by tupui**
Closes #18582

### CODE REVIEW

File: scipy/stats/_qmc.py [hunk 1]
```diff
@@ -1091,15 +1091,6 @@ class LatinHypercube(QMCEngine):
     ----------
     d : int
         Dimension of the parameter space.
-    centered : bool, optional
```
Comment (mdhaber) [inline scipy/stats/_qmc.py:1094]: `_random_cd` still references the `centered` keyword argument.

**Review by mdhaber — CHANGES_REQUESTED** [review]
One thing. Otherwise, if tests pass, this looks good.

File: scipy/stats/_qmc.py (hunk 1 above)
Comment (tupui) [reply scipy/stats/_qmc.py:1094]: Good catch thanks Matt

**Review by j-bowhay — APPROVED** [review]
Thanks @tupui LGTM provided CI is green

**Pull request merged**